docs/reference/presigned-url.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: default
+title: Presigned URL
+description: Configuring lakeFS to use presigned URLs.
+parent: Reference
+nav_order: 100
+has_children: false
+---
+
+# Presigned URL
+{: .no_toc }
+
+{% include toc.html %}
+
+lakeFS provides the option to access the data directly from the storage and not through lakeFS using presinged URL.
+lakeFS can generate a presigned URL to the object store by a user who has access to the object. The presigned URL can be used to access the object.
+lakeFS assume by default the object storage support presinged URLs, if not it should be disabled in the configuration.
```

Comment:
The presign support is enabled for block adapter that supports it, and can be disabled by the configuration. Note that the UI support is disabled by default.

Reply:
done